src/fpu/fpu_instructions.h

```diff
@@ -100,8 +102,24 @@ static void FPU_FPOP(void){
 	return;
 }
 
-static double FROUND(double in){
-	switch(fpu.round){
+static constexpr double RoundingEpsilon = 1e-14;
+
+// Determine if two numbers are equal "enough" based on an epsilon value.
+// Uses a dynamic adjustment based on the magnitude of the numbers.
+// Based on ideas from Bruce Dawson's blog post:
+// https://randomascii.wordpress.com/2012/02/25/comparing-floating-point-numbers-2012-edition/
+static bool AreEqualEnough(const double a, const double b,
```

Comment:
Static helper, so `lower_came_case`. This is generally useful, so can you move into `math_utils.h`? I also prefer the original name from the blog post, so how about `are_almost_equal_relative`? (I like having "relative" in the name, that's important)

Reply:
You got it.